### PR TITLE
Localize Sample App

### PR DESCRIPTION
**Pull request by daemoon**
Added localization support for sample site.

### CODE REVIEW

**Comment by petrsvihlik** [comment]
lgtm

File: DancingGoat/Localization/LanguageClient.cs [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        }
+
+        public Task<DeliveryItemResponse> GetItemAsync(string codename, IEnumerable<IQueryParameter> parameters)
+        {
+            return _client.GetItemAsync(codename, new[] { new LanguageParameter(_language) }.Concat(parameters));
+        }
+
+        public Task<DeliveryItemResponse<T>> GetItemAsync<T>(string codename, IEnumerable<IQueryParameter> parameters = null)
+        {
+            return parameters != null ? _client.GetItemAsync<T>(codename, new[] { new LanguageParameter(_language) }.Concat(parameters)) :
```
Comment (petrsvihlik) [inline DancingGoat/Localization/LanguageClient.cs]: missing null checks result in 
> Value cannot be null.
> Parameter name: second

can be seen on the homepage

Comment (daemoon) [inline DancingGoat/Localization/LanguageClient.cs]: This was caused by missing data in sample project. Should be fixed in next commit

**Comment by petrsvihlik** [comment]
some links don't work

**Pull request merged**